CRM/Odoosync/Upgrader.php

```diff
@@ -184,4 +221,85 @@ private function getSyncErrorTemplateWorkflowID() {
     return (int) $id;
   }
 
+  /**
+   * Enables all the option groups created by the extension
+   *
+   * @throws \CiviCRM_API3_Exception
+   */
+  private function enableExtensionOptionGroups() {
+    foreach ($this->optionGroups as $optionGroupName) {
+      $this->toggleOptionGroupActivity($optionGroupName, TRUE);
+    }
+  }
+
+  /**
+   * Enables all the custom groups created by the extension
+   *
+   * @throws \CiviCRM_API3_Exception
+   */
+  private function enableExtensionCustomGroups() {
+    foreach ($this->customGroups as $customGroupName) {
+      $this->toggleCustomGroupActivity($customGroupName, TRUE);
+    }
+  }
+
+  /**
+   * Disables all the custom groups created by the extension
+   *
+   * @throws \CiviCRM_API3_Exception
+   */
+  private function disableExtensionCustomGroups() {
+    foreach ($this->customGroups as $customGroupName) {
+      $this->toggleCustomGroupActivity($customGroupName, FALSE);
+    }
+  }
+
+  /**
+   * Disables all the option groups created by the extension
+   *
+   * @throws \CiviCRM_API3_Exception
+   */
+  private function disableExtensionOptionGroups() {
+    foreach ($this->optionGroups as $optionGroupName) {
+      $this->toggleOptionGroupActivity($optionGroupName, FALSE);
+    }
+  }
+
+  /**
+   * Sets is_active for OptionGroups
+   *
+   * @param string $name
+   * @param bool $isActive
+   *
+   * @throws \CiviCRM_API3_Exception
+   */
+  private function toggleOptionGroupActivity($name, $isActive) {
```

Comment:
since CiviCMR already has an entitiy called Activity, the name could be confusing, so I think in this case toogleOptionGroup would be enough, same for toggleCustomGroupActivity